phoenix-spark/src/it/scala/org/apache/phoenix/spark/AbstractPhoenixSparkIT.scala

```diff
@@ -50,7 +51,7 @@ class AbstractPhoenixSparkIT extends FunSuite with Matchers with BeforeAndAfter
   final val TenantId = "theTenant"
 
   var conn: Connection = _
-  var sc: SparkContext = _
+  var spark: SparkSession = _
```

Comment:
@twdsilva What is the difference between SparkContext and SparkSession?

Reply:
Is this more of an update to use new APIs?

Reply:
```SparkSession``` is the new unified entry point (see https://databricks.com/blog/2016/08/15/how-to-use-sparksession-in-apache-spark-2-0.html)